packages/astro/src/core/server-islands/vite-plugin-server-islands.ts

```diff
@@ -66,9 +80,8 @@ export function vitePluginServerIslands({ settings }: AstroPluginOptions): ViteP
 
 					// Append the name map, for prod
 					settings.serverIslandNameMap.set(comp.resolvedPath, name);
-
 					settings.serverIslandMap.set(name, () => {
-						return ssrEnvironment?.runner.import(comp.resolvedPath) as any;
+						return import(/* @vite-ignore */ comp.resolvedPath);
```

Comment:
Why importing from from the ssr environment doesn't work here?

Reply:
Because Cloudflare is not a runnable dev environment (meaning there's no runner)